docs/en/latest/apisix-variable.md

```diff
@@ -49,5 +49,6 @@ additional variables.
 | service_name        | core       | Name of Service.                                                                    |                |
 | redis_cmd_line      | Redis      | The content of Redis command.                                                       |                |
 | rpc_time            | xRPC       | Time spent at the rpc request level.                                                |                |
+| resp_body           | core       | This variable is only for logger and requires the logger to have a special configuration. |                |
```

Comment:
Which special configuration? I'm afraid that such a description is not useful for users to use this variable.

Reply:
I was update it